test/built-ins/Object/entries/exception-during-enumeration.js

```diff
@@ -2,8 +2,8 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
+id: sec-object.entries
```

Comment:
It actually makes sense to use section IDs in tests even at Stage 3 since the spec text should be near complete at that point, including the section IDs.

I talked to @bterlson and he is going to make sure that section IDs are correct in the spec starting at Stage 3 :smile: